ui/panels-plugin/src/plugins/time-series-chart/YAxisOptionsEditor.tsx

```diff
@@ -0,0 +1,87 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { TextField } from '@mui/material';
+import { OptionsEditorControl, OptionsEditorGroup, UnitSelector, UnitSelectorProps } from '@perses-dev/components';
+import { DEFAULT_UNIT, YAxisOptions, Y_AXIS_CONFIG } from './time-series-chart-model';
+
+export interface YAxisOptionsEditorProps {
+  value: YAxisOptions;
+  onChange: (yAxis: YAxisOptions) => void;
+}
+
+export function YAxisOptionsEditor({ value, onChange }: YAxisOptionsEditorProps) {
+  const handleUnitChange: UnitSelectorProps['onChange'] = (newUnit) => {
+    onChange({
+      ...value,
+      unit: newUnit,
+    });
+  };
```

Comment:
Not opposed to putting this inline like the other `onChange` handlers (but really it's fine however)